lighthouse-core/audits/byte-efficiency/unminified-javascript.js

```diff
@@ -77,21 +77,24 @@ class UnminifiedJavaScript extends ByteEfficiencyAudit {
     /** @type {Array<LH.Audit.ByteEfficiencyItem>} */
     const items = [];
     const warnings = [];
-    for (const requestId of Object.keys(artifacts.Scripts)) {
-      const scriptContent = artifacts.Scripts[requestId];
+    for (const {requestId, code} of artifacts.Scripts) {
+      if (!code) continue;
```

Comment:
should always be code, shouldn't there? Just excluding the zero-length case?

Reply:
The new main doc scripts I added already have a zero-length check, so all good there. But the previous code was doing a zero-length check, and I didn't want to modify that behavior.

we could add a zero length check to the other scripts too, and then remove this check.